AWS 9.3.9 with cluster-operator patch version (#488)
giantswarm/releases: aws/v9.3.9/README.md

# :zap: Giant Swarm Release v9.3.9 for AWS :zap:

**If you are upgrading from 9.3.8, upgrading to this release will not roll your nodes.**

> 

Reviewer: ```suggestion
```

> This is a patch release to update cluster-operator mitigating the unexpected deletion of app CRs in clusters. 

Reviewer: ```suggestion
This patch release fixes the problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters.

Please upgrade all older clusters to this version in order to prevent downtime. 
```

Reviewer: @tomahawk28 what will they prevent from upgrading to this release? Is it downtime? I am not sure of that part.

Author: The bug would delete coreDNS and other apps so  it would bring downtime to the tenant clusters. is that what you are asking? The change looks good so I'm going to commit your suggestion.

# :zap: Giant Swarm Release v9.3.9 for AWS :zap:

**If you are upgrading from 9.3.8, upgrading to this release will not roll your nodes.**

> This patch release fixes the problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters.

Reviewer 2: ```suggestion
This patch release fixes a problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters.
```

> Please upgrade all older clusters to this version in order to prevent downtime. 

Reviewer 2: ```suggestion
Please upgrade all older clusters to this version in order to prevent possible downtime. 
```

Maybe we should say possible downtime here? But we do need these clusters to be upgraded.

## Change details

### cluster-operator [0.23.17](https://github.com/giantswarm/cluster-operator/blob/legacy/CHANGELOG.md#02317---2020-10-19)

### Changed

- Get app-operator version from releases CR. 

> - Delete all chartconfig migration logic.

Reviewer 2: ```suggestion

### Fixed

- Remove all chartconfig migration logic that caused accidental deletion and is no longer needed.
```

I think this describes more clearly what we fixed. WDYT?

Author: Much better, thanks.
